omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestConstants.java

```diff
@@ -201,4 +201,10 @@
 	public static boolean SWAGGER_LOGS_ON = true;
 	
 	public static boolean SWAGGER_LOGS_OFF = false;
+	
+	public final static String PATIENT_PROGRAM_ATTRIBUTE_UUID = "3a2bdb18-6faa-11e0-8414-001e378eb67e";
+	
```

Comment:
For these, create a RestTestConstants2_2 just like this one: https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-2.1/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/RestTestConstants2_1.java

Reply:
Did you forget to revert changes in this class?

Reply:
Did you see the above?

Reply:
Sorry, let me revert the changes.